components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2880,10 +2880,11 @@ void RewardsServiceImpl::OnExternalWalletAuthorization(
   std::move(callback).Run(result, args);
 }
 
-void RewardsServiceImpl::ExternalWalletAuthorization(
-      const std::string& wallet_type,
-      const base::flat_map<std::string, std::string>& args,
-      ExternalWalletAuthorizationCallback callback) {
+void RewardsServiceImpl::OnDisconnectWalletsForAuthorization(
+    const std::string& wallet_type,
+    const base::flat_map<std::string, std::string>& args,
+    ExternalWalletAuthorizationCallback callback,
+    ledger::type::Result result) {
   if (!Connected()) {
```

Comment:
Do we need to check `result` here?

Reply:
Yes, good call. The question then becomes: what do we do when disconnecting fails?

Reply:
Yeah, I'm not sure there's much we can do other than log it for support purposes and move on (unless you're thinking showing the user some sort of UI would be useful, but I feel like that could happen down the line beyond this PR).

Reply:
I added a logged error.